### ISSUE TITLE
Novo evento(click) para datas inválidas

### ISSUE THREAD

**Issue by plovinicius** [nncl/ngx-datepicker#44]
**Sua sugestão está relacionada com um problema? Por favor descreva.**
- Hoje quando usamos data inválida o calendário não dispara o evento.

**Descreva a sugestão de solução**
- Deixar clicar em um uma data inválida e disparar um evento diferente, assim deixa uma liberdade pra quem está usando tomar decisões diferentes de acordo com o tipo de data que foi clicada (inválida e válida).

**Comment by nncl** [comment]
@plovinicius verdade, independentemente de termos controle via eventos `change` e `ngModelChange`, os dias inválidos não têm um handler fácil.

Vou implementar essa funcionalidade até a próxima semana :)

Enquanto isso, uma saída que possa ser interessante seria deixar todas as datas válidas, ou seja, sem passar nenhuma para o módulo, e num evento de change da model ou do componente você verifica se é válida ou não.
